cinema/tests/test_actor_api.py

```diff
@@ -16,9 +16,7 @@ def test_get_actors(self):
         response = self.client.get("/api/cinema/actors/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         actors_full_names = [actor["full_name"] for actor in response.data]
-        self.assertEqual(
-            sorted(actors_full_names), ["George Clooney", "Keanu Reeves"]
-        )
+        self.assertEqual(sorted(actors_full_names), ["George Clooney", "Keanu Reeves"])
```

Comment:
Ensure that the URL '/api/cinema/actors/' matches the actual URL pattern defined in your Django project. Hardcoding URLs can lead to maintenance issues if the URL patterns change. Consider using Django's reverse function to dynamically generate URLs based on the view names.